src/ol/format/feature.js

```diff
@@ -15,6 +15,7 @@ goog.require('ol.proj');
  * file formats.  See the documentation for each format for more details.
  *
  * @constructor
+ * @abstract
```

Comment:
Curious what makes this `@abstract`.  And if abstract, should the constructor do anything?